include/mapped_soa.h

```diff
@@ -85,6 +85,10 @@ class MappedSoA {
     key_to_index_.erase(key);
   }
 
+  void copy(size_t indexFrom, size_t indexTo) {
```

Comment:
nit: can we stick to underscore_lower_case_variable_naming? I'm following the google style guide :)
int underscore_names;
int unserscore_with_trailing_for_members_;
etc